ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java

```diff
@@ -253,18 +256,21 @@ protected WebView createViewInstance(ThemedReactContext reactContext) {
     reactContext.addLifecycleEventListener(webView);
     mWebViewConfig.configWebView(webView);
 
-    // BF changes
-    Log.w("BFWebView" + webView.getId(), "Initialize web view");
-    // webView.setLayerType(View.LAYER_TYPE_HARDWARE, null);
-    // webView.getSettings().setCacheMode(WebSettings.LOAD_NO_CACHE);
-    // end BF changes
     if (ReactBuildConfig.DEBUG && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
       WebView.setWebContentsDebuggingEnabled(true);
     }
 
     return webView;
   }
 
+  @Override
+  public boolean shouldOverrideUrlLoading(WebView view, String url) {
+    if (url.contains("bloomfire://")) {
+      return true;
+    }
+    return false;
```

Comment:
Consider `return url.contains("bloomfire://");` or even more performant if it works: `return url.startsWith("bloomfire://");`